composer.json

```diff
@@ -51,5 +51,11 @@
     "branch-alias": {
       "dev-master": "0.4.x-dev"
     }
-  }
+  },
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/dawehner/gastonjs.git"
```

Comment:
Well, we need to wait until the other PR, otherwise I could not test it locally to ensure it actually works